gatovid/models.py

```diff
@@ -64,19 +65,47 @@ def password(self) -> str:
 
     @password.setter
     def password(self, password: str) -> None:
-        self._password = bcrypt.generate_password_hash(password).decode("utf-8")
+        self._password = bcrypt.generate_password_hash(
+            password).decode("utf-8")
 
     def check_password(self, plaintext: str) -> bool:
         return bcrypt.check_password_hash(self.password, plaintext)
 
 
+class TokenBlacklist(db.Model):
+    """
+    Modelo para almacenar los tokens revocados.
+    """
+    __tablename__ = 'token_blacklist'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    token = db.Column(db.String(500), unique=True, nullable=False)
+    blacklisted_on = db.Column(db.DateTime, nullable=False)
+
+    def __init__(self, token):
+        self.token = token
+        self.blacklisted_on = datetime.datetime.now()
+
+    def __repr__(self):
+        return '<id: token: {}'.format(self.token)
+
+    @staticmethod
+    def check_blacklist(auth_token):
+        res = TokenBlacklist.query.filter_by(token=str(auth_token)).first()
+        if res:
```

Comment:
`return res`?

Reply:
ah no ya te entiendo. Igualmente puedes hacer `return res is None`

Reply:
Eso devolvería el objeto de la base de datos. Se podría poner como
```python
return res is not None
```
o algo así